File: 프로그래머스/0/120842. 2차원으로 만들기/2차원으로 만들기.cpp
#include <bits/stdc++.h>

using namespace std;

vector<vector<int>> solution(vector<int> num_list, int n) {
    int s = (int)num_list.size() / n;
    vector<vector<int>> answer(s, vector<int>(n));
    for(int i = 0; i < num_list.size(); i++) {
        answer[i / n][i % n] = num_list[i];
    }
    return answer;
}

File: 프로그래머스/0/120842. 2차원으로 만들기/2차원으로 만들기.swift
import Foundation

func solution(_ num_list:[Int], _ n:Int) -> [[Int]] {
    var answer: [[Int]] = []
    var a: [Int] = []
    for num in num_list {
        a.append(num)
        if a.count == n {
            answer.append(a)
            a = []
        }
    }
    return answer
}